doc/lua-filters.md

```diff
@@ -6,34 +6,19 @@ date: 'January 10, 2020'
 title: Pandoc Lua Filters
 ---
 
-# Introduction
+Create custom outputs with pandoc's embedded Lua engine.
 
-Pandoc has long supported filters, which allow the pandoc
-abstract syntax tree (AST) to be manipulated between the parsing
-and the writing phase. [Traditional pandoc
-filters](https://pandoc.org/filters.html) accept a JSON
-representation of the pandoc AST and produce an altered JSON
-representation of the AST. They may be written in any
-programming language, and invoked from pandoc using the
-`--filter` option.
-
-Although traditional filters are very flexible, they have a
-couple of disadvantages. First, there is some overhead in
-writing JSON to stdout and reading it from stdin (twice, once on
-each side of the filter). Second, whether a filter will work
-will depend on details of the user's environment. A filter may
-require an interpreter for a certain programming language to be
-available, as well as a library for manipulating the pandoc AST
-in JSON form. One cannot simply provide a filter that can be
-used by anyone who has a certain version of the pandoc
-executable.
-
-Starting with version 2.0, pandoc makes it possible to write
-filters in Lua without any external dependencies at all. A Lua
+## Introduction
+
+With Lua filters, you can write Pandoc filters  without any
+external dependencies. Besides the simpler set-up, Lua filters are
+generally faster and can access utility functions to manipulate
```

Comment:
faster than what?  Since the text mentioning JSON filters was removed, this is no longer clear.